integration_tests/snaps/godeps-with-go-packages/snapcraft.yaml

```diff
@@ -0,0 +1,14 @@
+name: test-package
```

Comment:
This doesn't seem to be using the dependencies.tsv and main.go copied in from `simple-godeps` (they're used in the new `godeps` snap instead). Should they be removed from here? Note also that the built .snap version of this was committed.